### ISSUE TITLE
Project Feedback

### ISSUE THREAD

**Issue by amanuel1995** [amanuel1995/MI-449-SS17-741-css-spacing-and-layout-T7bJIb#1]
@stuartpearman Can you take a look at this? It's [hosted here](https://goshuama-kapm87bvfi5cut7bjib.surge.sh/) and meets the following criteria:

- [x] Demonstrate an understanding of the CSS we've learned so far in the class
- [x] Use Flexbox to create the layout of your page
- [x] Use `margin` and `padding` to create space in and between elements
- [x] Use display `block` and `inline-block` properly
- [x] Have fun and do your best to make it pretty, the result should be something you would show to a future employer

**Comment by stuartpearman** [comment]
Coooool Amanuel. I like it.

I really like the background, but it makes it difficult to read the text. My eyes aren't too bad, but I don't think my mom would be able to read it.

Also, instead of putting side padding and margin on the bio paragraph, I would add padding to the container so that it lines up with the headings. The container should probably be a `div` rather than a `span`.

I had no idea you were raising funds for a startup, that's awesome! There are some opportunities to get funding in the area, Erik would be a great person to talk to about this.

Let me know when you've made changes :+1:

**Comment by amanuel1995** [comment]
You're right I wanted a binary background with a darker shade, I was lazy to go search 😺 . I will fix that.  And I'll also make that container a div and apply padding to it instead.

And, yeah I'm thinking about taking a risk back home. I generally gave up on searching for funds here because it's difficult to convince people to fund your overseas project while you and the funders are here. But thanks, I'll talk to Erik and see if there are hopes 😋  . I'll comment here or slack you once I make the changes.

**Comment by amanuel1995** [comment]
I have modified the background image and the text color a bit to make more readable while still looking fancy or weird 😅 .  I believe its more readable now.

And I have also made the style change that you suggested.

**Comment by stuartpearman** [comment]
Did you push to surge? I'm not seeing the changes

**Comment by amanuel1995** [comment]
Oh sorry, I forgot that. Got carried away in the live-server. Node is just too good 😃 . 
I just did push it to surge!

**Comment by stuartpearman** [comment]
Looks great Amanuel! :shipit:
